### PR TITLE
gemini: split up DEVICE_TITLE

### PR DESCRIPTION
**Pull request by mwarning**
Splits up DEVICE_TITLE into DEVICE_VENDOR, DEVICE_MODEL and DEVICE_VARIANT.

### CODE REVIEW

File: target/linux/gemini/image/Makefile [hunk 1]
```diff
@@ -140,7 +140,9 @@ GEMINI_NAS_PACKAGES:=kmod-md-mod kmod-md-linear kmod-md-multipath \
 #   This will be flashed from OpenWrt userland anyways so we only need
 #   the minimum to make the boot loader happy.
 define Device/dlink_dir-685
-	DEVICE_TITLE := D-Link DIR-685 Xtreme N Storage Router
+	DEVICE_VENDOR := D-Link
+	DEVICE_MODEL := DIR-685 Xtreme N Storage Router
+	DEVICE_VARIANT :=
```
Comment (dangowrt) [inline target/linux/gemini/image/Makefile]: Don't add empty fields

File: target/linux/gemini/image/Makefile [hunk 2]
```diff
@@ -152,7 +154,9 @@ endef
 TARGET_DEVICES += dlink_dir-685
 
 define Device/dlink_dns-313
-	DEVICE_TITLE := D-Link DNS-313 1-Bay Network Storage Enclosure
+	DEVICE_VENDOR := D-Link
+	DEVICE_MODEL := DNS-313
+	DEVICE_VARIANT := 1-Bay Network Storage Enclosure
```
Comment (dangowrt) [inline target/linux/gemini/image/Makefile]: This belongs to MODEL, it's not that there are multiple variants of that device.

**Review by dangowrt — CHANGES_REQUESTED** [review]
Two minor knits, apart from that looks good.

**Review by adschm — APPROVED** [review]
Looks good now.

**Comment by lynxis** [comment]
merged. Thank you!

**Pull request closed without merging**